lib/contentful/bootstrap/commands/create_space.rb

```diff
@@ -48,6 +48,7 @@ def fetch_space
               name: @space
             }
             options[:organization_id] = @token.read_organization_id unless @token.read_organization_id.nil?
+            management_client_init
```

Comment:
In https://github.com/contentful/contentful-bootstrap.rb/pull/55/files#diff-42edb6f486ee67a681d11b974db3cef0R75 you assign `client = management_client_init` shouldn't you do similar things here?